packages/titus-frontend/package.json

```diff
@@ -54,7 +54,6 @@
     "@storybook/react": "^6.4.9",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.0.0",
-    "chalk": "^5.0.0",
```

Comment:
is this not needed?

Reply:
I can't find it used directly in the frontend project. It is used by some of our devDependencies (like Jest) but at a lower version.

`depcheck` also flags it as unused.